### DISCUSSION TITLE
SQL Server Installation

### DISCUSSION THREAD

**Discussion by jamashie01** [dataplat/dbatools#9128] [Q&A]
I get the error below while I was trying to install SQL server, has anyone encountered such and what was the solution?

WARNING: [22:32:28][Resolve-DbaNetworkName] DNS name dbatoolslab not found | No such host is known
WARNING: [22:32:28][Install-DbaInstance] Failed to get reboot status from  | The argument is null or empty. Provide an argument that i
s not null or empty, and then try the command again.

**Comment by bilodeauj** [comment]
Did you ever figure out this issue with this? I was able to use install-dbaInstance on a standalone windows 10 pc, but when i tried this in a domain environment on server 2022, I got the same message.

**Reply by bilodeauj** [reply]
Just reread the Install-DbaInstance documentation https://docs.dbatools.io/Install-DbaInstance.html and figured out what i was doing wrong. The -SqlInstance parameter is the server name or server name \ named instance. So if you're passing dbatoolslab, you need to check and see if this is your server name or a named sql instance name. if it is in fact the name of your server, then name sure that your servername is properly showing up in dns and that it's resolvable by name
